code/app/global_variables.py

```diff
@@ -41,31 +134,155 @@
 
 
 for idx in np.arange(len(LGL_list)):
-	LGL_list[idx] = af.arith.cast(af.Array(LGL_list[idx]), af.Dtype.f64)
-	pass
+	LGL_list[idx] = np.array(LGL_list[idx], dtype = np.float64)
+	LGL_list[idx] = af.interop.np_to_af_array(LGL_list[idx])
+
+for idx in np.arange(len(gaussianNodesList)):
+	gaussianNodesList[idx] = np.array(gaussianNodesList[idx], \
+										dtype = np.float64)
+	gaussianNodesList[idx] = af.interop.np_to_af_array(gaussianNodesList[idx])
 
-N_LGL       = 16
-xi_LGL      = None
-lBasisArray = None
+x_nodes         = af.interop.np_to_af_array(np.array([-1., 1.]))
+N_LGL           = 16
+xi_LGL          = None
+lBasisArray     = None
+lobatto_weights = None
+N_Elements      = None
+element_nodes   = None
+u               = None
+time            = None
+c               = None
 
-def populateGlobalVariables(N = 16):
+
+def populateGlobalVariables(Number_of_LGL_pts = 8, Number_of_elements = 10):
 	'''
-	Initialize the global variables
-	
+	Initialize the global variables.
 	Parameters
 	----------
-	N : int
-		Number of LGL points.
+	N_LGL : int
+			Number of LGL points.
+			Declares the number and the value of
```

Comment:
Why is there an incomplete Statement here?
Please review all the doc-strings before doing your next pull request.